Upstream-Status: Backport

* Backported from the ToT: https://crrev.com/c/1360275

Signed-off-by: Maksim Sisov <msisov@igalia.com>
---
From e96597372969dc31ae86f7d3e08af93be45ed617 Mon Sep 17 00:00:00 2001
From: Maksim Sisov <msisov@igalia.com>
Date: Tue, 4 Dec 2018 08:40:20 +0000
Subject: [PATCH 01/16] fixup! [ozone] move gbm wrapper into a separate source
 set.

Add accidentelly removed libdrm dep.

Fixes the following compilation error:
In file included from ../../ui/ozone/common/linux/gbm_wrapper.cc:9:
../../build/linux/debian_sid_amd64-sysroot/usr/include/xf86drm.h:40:10: fatal error: 'drm.h' file not found
         ^~~~~~~

TBR=spang@chromium.org

Change-Id: I8fde77a2c5b7d98c8dc70ba1b34a7766b047dd21
Reviewed-on: https://chromium-review.googlesource.com/c/1360275
Reviewed-by: Maksim Sisov <msisov@igalia.com>
Commit-Queue: Maksim Sisov <msisov@igalia.com>
Cr-Commit-Position: refs/heads/master@{#613473}
---
 ui/ozone/common/linux/BUILD.gn | 1 +
 1 file changed, 1 insertion(+)

diff --git a/ui/ozone/common/linux/BUILD.gn b/ui/ozone/common/linux/BUILD.gn
index 2bf3481ffe87..c808a515e148 100644
--- a/ui/ozone/common/linux/BUILD.gn
+++ b/ui/ozone/common/linux/BUILD.gn
@@ -30,6 +30,7 @@ source_set("gbm") {
   deps = [
     ":drm",
     "//base:base",
+    "//build/config/linux/libdrm",
     "//third_party/minigbm",
     "//ui/gfx:buffer_types",
     "//ui/gfx:memory_buffer",
-- 
2.17.1

